Mobile_Detect.php

```diff
@@ -94,13 +94,13 @@ class Mobile_Detect
      * HTTP headers in the PHP-flavor. So HTTP_USER_AGENT and SERVER_SOFTWARE.
      * @var array
      */
-    protected $httpHeaders = array();
+    protected $httpHeaders = [];
```

Comment:
Please keep this file compatible to PHP 5.3 an use the long array syntax.

Reply:
Ok, will do the changes as required to support PHP 5.3